go/v1/resource_descriptor.go

```diff
@@ -0,0 +1,14 @@
+/*
+Wrapper APIs for in-toto attestation ResourceDescriptor protos.
+*/
+
+package v1
+
+func (d *ResourceDescriptor) Validate() bool {
+	// at least one of name, URI or digest are required
+	if d.GetName() == "" && d.GetUri() == "" && len(d.GetDigest()) == 0 {
+		return false
```

Comment:
Would it make sense, now or in a future version, to have this return an error indicating why it failed?  Perhaps this could return `error` instead of bool?

Reply:
Yeah, I think this makes total sense, and I'd rather change the function signature now, than later :)